src/scalper.ts

```diff
@@ -254,16 +306,18 @@ export class Scalper {
       )
     )[0];
 
+    await this.cancelStaleOrders(mangoAccount, mangoGroup, perpMarket);
```

Comment:
Add a comment explaining why this makes it safe for having multiple threads running the gammaScalp at once, or a TODO to make this function handle that.